antlr4/org/eclipse/jnosql/query/grammar/Query.g4

```diff
@@ -24,7 +24,7 @@ lt: not? name '<' value;
 lte: not? name '<=' value;
 between: not? name 'between' value 'and' value;
 in: name not? 'in' '('value (',' value)* ')';
-like: name not? 'like' string;
+like: name not? 'like' string_parameter ;
```

Comment:
Why not use "value" here?

Reply:
Because "value" supports numbers, arrays, functions, and jsons too... of course, the 'parameter' type doesn't provide guarantees to support only string values... but it prevents developers to be putting these unsupported data types.

Reply:
Yeap I saw the next lines.
It makes sense, thank you Max.